src/main/java/com/hit/OneOrZeroKnapsackAlgoImpl.java

```diff
@@ -1,17 +1,17 @@
-package main.java.com.hit;
+package com.hit;
 
 import java.util.ArrayList;
 import java.util.Collection;
 
-public class OneOrZeroKnapsackAlgoImpl implements IAlgoKnapsack {
+public class OneOrZeroKnapsackAlgoImpl implements main.java.com.hit.IAlgoKnapsack {
```

Comment:
Why there is a change here with main.java.com.hit.IAlgoKnapsack ?
it should be :
public class OneOrZeroKnapsackAlgoImpl implements IAlgoKnapsack